packages/chart/src/ChartUtils.ts

```diff
@@ -1874,25 +1874,56 @@ class ChartUtils {
 
   makeDefaultLayout(theme: ChartTheme): Partial<Layout> {
     const { dh } = this;
+
```

Comment:
Can you add a jsdoc to this method and include a link to https://plotly.com/javascript/reference/layout/ in it? It doesn't include everything (they're spread under different pages in the sidebar), but gives the starting point at least